### PR TITLE
Deprecate HHVM workload and Facebook OSS

### PR DESCRIPTION
**Pull request by Xusine1131**

### CODE REVIEW

File: benchmarks/fb-oss-performance/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This image is deprecated in CloudSuite 4 and not maintained by us anymore. However, you can still pull the binary image if you want to run the workload.
```
Comment (aansaarii) [inline benchmarks/fb-oss-performance/README.md:1]: not maintained --> will not be maintained or we will not maintain.

File: commons/hhvm/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This image is deprecated in CloudSuite 4 and not maintained by us anymore. However, you can still pull the binary image if you want to run the workload.
```
Comment (aansaarii) [inline commons/hhvm/README.md:1]: not maintained --> will not be maintained or we will not maintain.

File: docs/benchmarks/fb-oss-performance.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Facebook Workload
 
+> **Warning:** This benchmark is deprecated in CloudSuite 4 and not maintained anymore. However, you can still download the binary image with the command in this document and run the experiment.
```
Comment (aansaarii) [inline docs/benchmarks/fb-oss-performance.md:3]: not maintained --> will not be maintained or we will not maintain.
command --> commands

File: .github/workflows/build-images.yaml [hunk 4]
```diff
@@ -121,32 +121,32 @@ jobs:
...
+  #     - if: ${{ needs.base-os.outputs.is_parent_modified == 'true' }}
+  #       run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
+  #     - name: build and push
+  #       run: "./.github/scripts/build-images.sh"
+  #       env:
+  #         IMG_TAG: "${{ matrix.tag }}"
+  #         DF_PATH: "./commons/${{ github.job }}/${{ matrix.tag }}"
+  #         DBX_PLATFORM: ${{ matrix.platform }}
+  #     - id: set_is_parent_modified
+  #       run: echo "::set-output name=is_parent_modified::${MODIFIED}"
```
Comment (aansaarii) [inline .github/workflows/build-images.yaml:149]: If we deprecate the workload, do we need to keep these lines as comments in the script? It makes it ugly and does not serve any purpose if we won't build them again in the future.

File: .github/workflows/build-images.yaml [hunk 5]
```diff
@@ -411,30 +411,30 @@ jobs:
...
+  #     - if: ${{ needs.mysql.outputs.is_parent_modified == 'true' ||
+  #         needs.siege.outputs.is_parent_modified == 'true' ||
+  #         needs.hhvm.outputs.is_parent_modified == 'true' }}
+  #       run: echo "IS_PARENT_MODIFIED=true" >> $GITHUB_ENV
+  #     - name: build and push
+  #       run: "./.github/scripts/build-images.sh"
+  #       env:
+  #         IMG_TAG: "${{ matrix.tag }}"
+  #         DF_PATH: "./benchmarks/${{ github.job }}/${{ matrix.tag }}"
+  #         DBX_PLATFORM: ${{ matrix.platform }}
```
Comment (aansaarii) [inline .github/workflows/build-images.yaml:437]: If we deprecate the workload, do we need to keep these lines as comments in the script? It makes it ugly and does not serve any purpose if we won't build them again in the future.

**Comment by aansaarii** [comment]
I also suggest that we remove this workload completely from the main branch as everything on the main is considered as supported cloudsuite workloads. We can devote a branch that still holds fb-oss for anyone who wants to have access to it.

**Review by UlisesLuzius — APPROVED** [review]

**Pull request merged**